packages/mjml-hero/README.md

```diff
@@ -1,5 +1,21 @@
 ## mj-hero
 
+This element displays a hero image in the background of an email.
+It behaves like an `mj-section` with a single `mj-column`.
+
+Set `background-height` and `background-width`
+  to the actual height and width of the image identified in
+  `background-url`.
+
+In both fixed and fluid modes, it's best to use an image with width
+  the same as the `mj-body` width (`width="600px"` by default).
+Further, it's best to use an image with height the same or larger
+  than the `height` of `mj-hero`.
+Image sizes outside this guidance often result in poor rendering.
+
+Use `background-color` to provide a fallback color
+  in case an email client doesn't support `background-url`.
+
```

Comment:
Those use to be warnings with aside

Reply:
True. I'll say "asides" and sidebars work better when they're supplementing the existing text. We didn't have any. Once I looked at it, I hoped you'd feel this material makes fine text.